sig-storage/volume-plugin-faq.md

```diff
@@ -12,11 +12,12 @@ A *Kubernetes Volume plugin* extends the Kubernetes volume interface to support
 **How do I implement a Volume plugin?**
 
 There are three methods to implement a volume plugin:
-1. In-tree volume plugin
-2. Out-of-tree FlexVolume driver
+1. In-tree volume plugin [deprecated]
```

Comment:
do we need to make an announcement in the release notes that it's deprecated?

Reply:
The existing drivers will continue to work and are not deprecated. Adding new volumes is deprecated, and this is documenting that.